e2e/scripts/st_file_uploader.py

```diff
@@ -14,14 +14,17 @@
 
 import streamlit as st
 
-
+st.set_option("deprecation.v00_063_showfileUploaderEncoding", True)
 result = st.file_uploader("Drop a file:", type=["txt"])
 # result = st.file_uploader("Drop a file:", type=["txt"], accept_multiple_files=False)
 if result is not None:
     st.text(result.getvalue())
 else:
     st.text("No upload")
 
+st.set_option("deprecation.v00_063_showfileUploaderEncoding", False)
+st.file_uploader("Disable deprecation", type=["txt"], disable_deprecation_warning=True)
```

Comment:
I'm confused... why do we have two methods of enabling/disabling warnings?

Reply:
That's a mistake on my side. I had implemented it at the API level prior and forgot to clean this up.